packages/screens/RiotGame/component/RipperSelectorModalV2.tsx

```diff
@@ -21,7 +21,7 @@ import FlexRow from "../../../components/FlexRow";
 import { SVG } from "../../../components/SVG";
 import { TertiaryBox } from "../../../components/boxes/TertiaryBox";
 import { SpacerRow } from "../../../components/spacer";
-import { getRipperRarity } from "../../../utils/game";
```

Comment:
maybe find a more explicit name for this file

Reply:
OK changed it => RipperGridSelectorModal